### PR TITLE
feat: add brief section

### PR DESCRIPTION
**Pull request by birkaany**
Brief section developed;
- BriefSection component has been created by using mobile-first approach
- Specified to this component classes added via BriefSection.module.css
- Added icons via Icon component's json file

Important note: "Title" component's style folder has been manipulated to style BriefSection's heading elements, this needs to be fixed/improved.

### CODE REVIEW

File: components/Title/Title.module.css [hunk 1]
```diff
@@ -1,3 +1,23 @@
 .title {
-  @apply; /* TODO <-- add default title styles here */
+  @apply text-6xl; /* TODO <-- add default title styles here */
```
Comment (eraygundogmus) [inline components/Title/Title.module.css]: can you remove this comment?

File: components/BriefSection/BriefSection.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import style from "./BriefSection.module.css";
+import Icon from "../Icon/Icon";
+import Title from "../Title/Title";
+
+const BriefSection = () => {
+  const cardArray: any = [
```
Comment (eraygundogmus) [inline components/BriefSection/BriefSection.tsx]: can you create a sample.json and create this array on there? then you can import it here and i believe code will be cleaner on this component

**Review by eraygundogmus — CHANGES_REQUESTED** [review]

File: components/BriefSection/BriefSection.tsx (hunk 2 above)
Comment (birkaany) [reply components/BriefSection/BriefSection.tsx]: I created a new json to storage data and exported into component

File: components/Title/Title.module.css (hunk 1 above)
Comment (birkaany) [reply components/Title/Title.module.css]: Removed unnecessary comment lines

**Review by eraygundogmus — COMMENTED** [review]
Thanks for your contribution

**Pull request merged**